app/src/androidTest/java/org/mozilla/fenix/ui/robots/HistoryRobot.kt

```diff
@@ -166,3 +177,27 @@ private fun snackBarUndoButton() = onView(withId(R.id.snackbar_btn))
 
 private fun assertUndoDeleteSnackBarButton() =
     snackBarUndoButton().check(matches(withText("UNDO")))
+
+private fun deleteAllHistoryPromptTitle() =
```

Comment:
Nit: drop "all" if it doesn't make a difference,  to keep method names shorter

Reply:
Done !